docker-compose.pinner.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  pinner:
+    # uncomment "build" and comment out "image" to build from sources
+    # build: https://github.com/SkynetLabs/pinner.git#main
+    image: skynetlabs/pinner
```

Comment:
we should use a tagged version of pinner from the beginning

Reply:
That's great but we don't have one. It seems we'll block on this until Pinner is ready for prod.